scripts/main.js

```diff
@@ -14,7 +14,10 @@ const LEGIS_LIMIT = 1000;
 /** constant {object} */
 var CC_API_TOKENS = null;
 
-/** 
+// Labels
+const COLUMN_NAME_PARTY = 'PARTY'
```

Comment:
Good idea, constants for all the column names would probably be a good idea if we can get around to it.